content/pt-br/docs/reference/issues-security/issues.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Rastreador de Issue Kubernetes
+weight: 10
+aliases: [/cve/,/cves/]
+---
+
+Para reportar uma issue de segurança, sigua [processo de divulgação de issues do Kubernetes](/docs/reference/issues-security/security/#report-a-vulnerability).
+
+O trabalho no código do Kubernetes e os problemas públicos podem ser encontrados usando [GitHub Issues](https://github.com/kubernetes/kubernetes/issues/).
```

Comment:
On line `7` you use `issue de segurança` and in line `9`  you use `problemas públicos` , should be good to keep a standard and use the same work since they're kind the same thing.